index.html

```diff
@@ -1,70 +1,218 @@
 <!DOCTYPE html>
 <html lang="en">
-      <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
-        <link rel="stylesheet" href="styles/main.css">
-        <title>JT's Portfolio</title>
-      </head>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="stylesheet" href="styles/main.css">
+    <title>JT's Portfolio</title>
+  </head>
 <body>
   
-  <header>
-    <div class="logo">
-      <a href="#">
-        <!-- <img src="images/icon.png" alt="Jt-logo"> -->
-        My Logo
-      </a>
-      
-    </div>
-
-    <div>
-      <nav>
-        <a href="#" class = "menu">
-        <img src="images/menu.png" alt="Menu">
+  <div class="grid-container">
+  
+    <header>
+      <div class="logo">
+        <a href="#">
+          <!-- <img src="images/icon.png" alt="Jt-logo"> -->
+          My Logo
         </a>
-     </nav>
-    </div>
-  </header>
+      </div>
+      <div>
+        <nav>
+          <a href="#" class = "menu">
+            <img src="images/menu.png" alt="Menu">
+          </a>
+        </nav>
+      </div>
+    </header>
   
-  <div class="main">
-
-    <h1>I'm Jud&aacute; <br> Glad to see <br> you!</h1>
+    <div class="main" id="bg-main">
+      <div class="content">
+        <h1>I'm Jud&aacute; <br> Glad to see <br> you!</h1>
+        <p>
+          I'm a software developer! I can help you 
+          build a product, feature or website Look 
+          through some of my work and 
+          experience! If you like what you see and 
+          have a project you need coded, don't 
+          hesitate to contact me.
+        </p>
+      </div>
+      <div class="socials">
+        <h4>LET'S CONNECT</h4>
+        <ul class="icon-bar">
+          <li><a href="#">
+            <img src="images/twitter.png" alt="Twitter">
+          </a></li>
+          <li><a href="#">
+            <img src="images/linkedin.png" alt="Linkedin">
+          </a></li>
+          <li><a href="#">
+            <img src="images/facebook.png" alt="Facebook">
+          </a></li>
+          <li><a href="#">
+            <img src="images/github.png" alt="Github">
+          </a></li>
+          <li><a href="#">
+            <img src="images/instagram.png" alt="Facebook">
+          </a></li>
+        </ul>
+      </div>
+    </div> <!-- End of MAIN -->
     
-    <div class="content">
-    <p>
-      I'm a software developer! I can help you 
-      build a product, feature or website Look 
-      through some of my work and 
-      experience! If you like what you see and 
-      have a project you need coded, don't 
-      hesitate to contact me.
-      </p>
+    <div class="works"> <!-- WORKS -->
```

Comment:
- [ ] As per project requirements, the container of the section is meant to be a `<section>` tag. Implementing this will improve the overall readability of the code while encouraging the use of semantic HTML tags. 😄

- [ ] To display the projects in the work section, how about we use `display:grid`? Applying CSS grid, will keep the items or projects displayed consistently in a simple way.